Easydict/NewApp/View/SettingView/Tabs/View/Shortcut/KeyHolderWrapper.swift

```diff
@@ -64,17 +66,13 @@ extension KeyHolderWrapper {
         func recordViewDidEndRecording(_: RecordView) {}
 
         func recordView(_ recordView: RecordView, didChangeKeyCombo keyCombo: KeyCombo?) {
+            if keyCombo == nil { // clear shortcut
+                Shortcut.shared.updateMenu(type)
+            }
             if let key = keyCombo {
                 // shortcut validate confict
                 if Shortcut.validateShortcut(key) {
-                    if #available(macOS 12, *) {
                         confictAlterMessage = ShortcutConfictAlertMessage(title: String(localized: "shortcut_confict_title \(keyCombo!.keyEquivalentModifierMaskString + keyCombo!.characters)"), message: String(localized: "shortcut_confict_message \(Shortcut.shared.confictMenuItem?.title ?? "")"))
```

Comment:
The indent of this line is not adjusted correctly. And please make this line shorter if possible, like define `title` and `message` in a separate line.

Reply:
Use `key` instead of `keyCombo!`.